_articles/how-to-contribute.md

```diff
@@ -216,8 +216,12 @@ Open source isn't an exclusive club; it's made by people just like you. "Open so
 
 You might scan a README and find a broken link or a typo. Or you're a new user and you noticed something is broken, or an issue that you think should really be in the documentation. Instead of ignoring it and moving on, or asking someone else to fix it, see whether you can help out by pitching in. That's what open source is all about!
 
+If you are a beginner in open source you can work on a good first issue. Good first issue is a feature GitHub ships for encouraging new and begineer open source contributors for contributing in an open source project. You can find such issues in [contribute pages](https://github.com/prettier/prettier/contribute).
```

Comment:
It looks like you forgot to delete this.

Reply:
Ref https://github.com/github/opensource.guide/pull/1110#discussion_r327692279.

Reply:
I'm not sure what you're referring to in that comment. You added the paragraph I suggested below, but didn't delete this one.

Reply:
I see, right, misunderstanding in continuity. Done.